modules/gathering-clusterversion-history.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-clusterversion-history_{context}"]
+= Gathering ClusterVersion history 
+
+When updating your cluster, the cluster version operator (CVO) creates a `status.history` entry to record the update. This can be useful to determine if history-related bugs apply to your cluster. The entry can also reveal correlation between changes in cluster behavior with potential triggers, although correlation does not imply causation. 
+
+[NOTE]
+====
+The ClusterVersion history has a size limit, so in cases where enough updates occurred to have reached that limit, the least relevant entries in the history are pruned to accommodate the limit. 
+====
+
+It is possible to view the ClusterVersion history though the web console and through the CLI.
```

Comment:
```suggestion
It is possible to view the ClusterVersion history in the {product-title} web console, or by using the {oc-first}.
```
Per the IBMSG, do not use "through" to mean "by using":
![image](https://github.com/openshift/openshift-docs/assets/40172997/8f075877-6a24-4ab2-874f-22d9f0d4ee3f)

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#through

Reply:
Also suggest using the attributes here for the web console and CLI names